python/datasources/covid_tracking_project_metadata.py

```diff
@@ -10,3 +11,48 @@ def get_id():
     @staticmethod
     def get_table_name():
         return 'covid_tracking_project_metadata'
+
+    def upload_to_gcs(self, _, **attrs):
+        raise NotImplementedError(
+            'upload_to_gcs should not be called for CtpMetadata')
+
+    def write_to_bq(self, dataset, gcs_bucket, **attrs):
+        gcs_file = self.get_attr(attrs, 'filename')
+
+        # Download the raw data
+        df = gcs_to_bq_util.load_csv_as_dataframe(gcs_bucket, gcs_file)
+        self.clean_frame_column_names(df)
+
+        # Standardize the data
+        keep_cols = [
+            'state_postal_abbreviation', 'api_death',
+            'defines_other_death', 'race_ethnicity_separately_death',
+            'race_ethnicity_combined_death', 'race_mutually_exclusive_death',
+            'combined_category_other_than_api_death', 'race_death',
+            'ethnicity_death', 'api_cases', 'defines_other_cases',
```

Comment:
They only have these for deaths and cases I take it? Maybe add a comment about why there isn't anything for hospitalizations or tests

Reply:
Yeah, they didn't provide us with any info on tests or hospitalizations, unfortunately.